build.sbt

```diff
@@ -56,6 +56,29 @@ lazy val config = project
   .settings(moduleName := "frees-rpc-config")
   .settings(configSettings)
 
+lazy val interceptors = project
+  .in(file("modules/interceptors"))
+  .settings(moduleName := "frees-rpc-interceptors")
+  .settings(interceptorsSettings)
+
+lazy val `prometheus-shared` = project
+  .in(file("modules/prometheus/shared"))
+  .dependsOn(interceptors % "compile->compile;test->test")
+  .settings(moduleName := "frees-rpc-prometheus-shared")
+  .settings(prometheusSettings)
+
+lazy val `prometheus-server` = project
+  .in(file("modules/prometheus/server"))
+  .dependsOn(`prometheus-shared` % "compile->compile;test->test")
+  .dependsOn(server % "compile->compile;test->test")
```

Comment:
`prometheus-server` only depends on `server` for testing, right? Or the idea is to include `prometheus-server` dependency instead of `server`?

Reply:
Not only for tests, you need `server` to make it work with `prometheus-server`, that's why it should be added transitively. `prometheus-server` doesn't make sense without `server`.